### PR TITLE
modify migration

### PR DESCRIPTION
**Pull request by emya**
Part of [#146656037]

### CODE REVIEW

File: lib/etl/cli/cmd/migration.rb [hunk 1]
```diff
@@ -102,39 +109,27 @@ def create_migration(up, down)
 
       def up_sql
         t = ETL::Redshift::Table.new(table)
+        puts "schema_map #{schema_map}"
```
Comment (ChrisPRobinson) [inline lib/etl/cli/cmd/migration.rb]: Add a log statement instead of puts?

File: lib/etl/cli/cmd/migration.rb [hunk 2]
```diff
@@ -102,39 +109,27 @@ def create_migration(up, down)
...
-            t.smallint(key.to_sym) if type.to_s.start_with? "tinyint"
+          else
+            define_type(t, key, type)
           end
         end
 
         primary_keys.each { |pk| t.add_primarykey(pk) }
         up = <<END
-        @client.execute("#{t.create_table_sql}")
+        @client.execute("#{t.create_table_sql.gsub!('"', '\"')}")
```
Comment (ChrisPRobinson) [inline lib/etl/cli/cmd/migration.rb]: Why is this change needed?

Comment (ChrisPRobinson) [inline lib/etl/cli/cmd/migration.rb]: If this change is truly needed it should go into the redshift client instead.

Comment (emya) [inline lib/etl/cli/cmd/migration.rb]: We need to use double quotes for column names, and we use double quote for whole query. So in each query, we have to replace `"` used for column names with `\"`

Comment (ChrisPRobinson) [inline lib/etl/cli/cmd/migration.rb]: I think we need to fix the sql generation so it generates the correct sql rather than fixing things afterwards.

File: lib/etl/cli/cmd/migration.rb (hunk 1 above)
Comment (emya) [inline lib/etl/cli/cmd/migration.rb]: I forgot to remove puts. This is migration command, not part of etl job. Let me get rid of this.

File: lib/etl/cli/cmd/migration.rb (hunk 2 above)
Comment (emya) [inline lib/etl/cli/cmd/migration.rb]: I think this change is specific to the migration file, and we don't need this process for creating  usual query.

Comment (ChrisPRobinson) [inline lib/etl/cli/cmd/migration.rb]: Understood now, as we discussed a heredoc might be best.
